airgun/views/virtwho_configure.py

```diff
@@ -0,0 +1,150 @@
+from widgetastic_patternfly import BreadCrumb
+
+from widgetastic.widget import (
+    Checkbox,
+    ConditionalSwitchableView,
+    Table,
+    Text,
+    TextInput,
+    View,
+)
+from airgun.views.common import (
+    BaseLoggedInView,
+    SearchableViewMixin,
+    SatTab,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    FilteredDropdown,
+)
+
+
+class VirtwhoConfiguresView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Virt-who Configurations']")
+    new = Text("//a[contains(@href, '/foreman_virt_who_configure/configs/new')]")
+    table = Table(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class VirtwhoConfigureCreateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    name = TextInput(id='foreman_virt_who_configure_config_name')
+    interval = FilteredDropdown(id='foreman_virt_who_configure_config_interval')
+    satellite_url = TextInput(id='foreman_virt_who_configure_config_satellite_url')
+    hypervisor_id = FilteredDropdown(id='foreman_virt_who_configure_config_hypervisor_id')
+    debug = Checkbox(id='foreman_virt_who_configure_config_debug')
+    proxy = TextInput(id='foreman_virt_who_configure_config_proxy')
+    no_proxy = TextInput(id='foreman_virt_who_configure_config_no_proxy')
+    filtering = FilteredDropdown(id='foreman_virt_who_configure_config_listing_mode')
+    filtering_content = ConditionalSwitchableView(reference='filtering')
+    hypervisor_type = FilteredDropdown(id='foreman_virt_who_configure_config_hypervisor_type')
+    hypervisor_content = ConditionalSwitchableView(reference='hypervisor_type')
+    submit = Text('//input[@name="commit"]')
```

Comment:
well, I'm not sure about UI. Does that not have a button tag?

Reply:
Looking at our code, seems that Satellite is not very consistent - half of Submits are `<button>` and half are `<input>`. This one happens to be `<input>`.

Reply:
OK

Reply:
<input type="submit" name="commit" value="Submit" class="btn btn-primary remove_form_templates" data-disable-with="Submit" />